staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/BUILD

```diff
@@ -100,9 +94,7 @@ go_library(
 go_test(
     name = "go_default_test",
     srcs = ["customresource_handler_test.go"],
-    data = [
-        "//api/openapi-spec",
-    ],
+    data = glob(["testdata/**"]),
```

Comment:
Since we're not going toward more bazel, AFAIK, it's probably not important, but do we need to depend on everything here?

Reply:
this was autogenerated, so... /shrug ?